### DISCUSSION TITLE
MEETING!

### DISCUSSION THREAD

**Discussion by LucyKershaw** [OSIPI/DCE-DSC-MRI_CodeCollection#119] [General]
Hello all,

We need to have a meeting to discuss the roadmap and our plans for the coming year - I'm aware that our timezones might make this tricky but who would be available for a meeting at 11am GMT (10pm Sydney, noon European, sadly 6am East coast USA) on Thursday next week?

Lucy

**Comment by plaresmedima** [comment]
I can do that

**Reply by petravanhoudt** [reply]
would work for me as well
